Tabby/app/library/[category]/index.tsx

```diff
@@ -52,18 +54,32 @@ const CategoryPage: React.FC = () => {
             <FavoriteButtonIcon isFavorite={book.isFavorite} />
         </Pressable>
     );
+    
+    const renderItem = ({item}: {item: Book}) => {
+        console.log("Search: ", search.length);
```

Comment:
Can we add comments here just to remind ourselves to remove the log stuff in production build.
A simple 
// DEBUG 
or something would do